golden/bullseye.pkr.hcl

```diff
@@ -0,0 +1,50 @@
+variables {
```

Comment:
I don't recall how variables work in HCL. I really am fond of packer as well as HCL, although if wr aren't sure it works, can you remove it? These variables can be removed for now, I believe. They can always be reintroduced.